packages/data-service-generator/src/server/static/src/providers/secrets/base/secretsManager.service.base.spec.ts

```diff
@@ -31,7 +31,7 @@ describe("Testing the secrets manager base class", () => {
   it("should throw error if dont get key", () => {
     return expect(secretsManagerServiceBase.getSecret("")).rejects.toThrow();
   });
-  it("should throw an exeption if getting null key", () => {
+  it("should throw an exception if getting null key", () => {
```

Comment:
Please revert this single change and then the matching changes in all the *.snap files - we wish to exclude the changes to the generated code from this PR. You can open a new issue and submit a PR just for this change separately (with the matching *.snap files)